test/setup/iam.tf

```diff
@@ -20,7 +20,9 @@ locals {
     "roles/artifactregistry.admin",
     "roles/documentai.editor",
     "roles/firebase.admin",
+    "roles/iam.serviceAccountAdmin",
```

Comment:
This address an error seen in integration tests:
```
Step #2 - "apply": Error: Error creating service account: googleapi: Error 403: Permission 'iam.serviceAccounts.create' denied on resource (or it may not exist).
Step #2 - "apply": Details:
Step #2 - "apply": [
Step #2 - "apply":   {
Step #2 - "apply":     "@type": "type.googleapis.com/google.rpc.ErrorInfo",
Step #2 - "apply":     "domain": "iam.googleapis.com",
Step #2 - "apply":     "metadata": {
Step #2 - "apply":       "permission": "iam.serviceAccounts.create"
Step #2 - "apply":     },
Step #2 - "apply":     "reason": "IAM_PERMISSION_DENIED"
Step #2 - "apply":   }

```